convert2rhel/systeminfo.py

```diff
@@ -60,22 +72,15 @@ def __init__(self):
         self.name = None
         # Single-word lowercase identificator of the system (e.g. oracle)
         self.id = None  # pylint: disable=C0103
+        # The optional last part of the distribution name in brackets: "... (Core)" or "... (Oopta)"
+        self.distribution_id = None
         # Major and minor version of the operating system (e.g. version.major == 8, version.minor == 7)
         self.version = None
         # Platform architecture
         self.arch = None
         # Fingerprints of the original operating system GPG keys
         self.fingerprints_orig_os = None
-        # Fingerprints of RHEL GPG keys available at:
-        #  https://access.redhat.com/security/team/key/
-        self.fingerprints_rhel = [
-            # RPM-GPG-KEY-redhat-release
-            "199e2f91fd431d51",
-            # RPM-GPG-KEY-redhat-legacy-release
-            "5326810137017186",
-            # RPM-GPG-KEY-redhat-legacy-former
-            "219180cddb42a60e",
-        ]
+        self.fingerprints_rhel = RHEL_FINGERPRINTS
```

Comment:
I would just get rid of this attribute as the module level constant is a valid replacement.

Reply:
There seems to not have a lot of places calling this property, so +1 to this suggestion.

```bash
(.venv3) [r0x0d@fedora convert2rhel]$ rg fingerprints_rhel
convert2rhel/systeminfo.py
71:        self.fingerprints_rhel = [

convert2rhel/pkghandler.py
339:        system_info.fingerprints_orig_os + system_info.fingerprints_rhel
590:    non_red_hat_pkgs = get_installed_pkgs_w_different_fingerprint(system_info.fingerprints_rhel)
641:        pkg_objects = get_installed_pkgs_w_different_fingerprint(system_info.fingerprints_rhel, pkg)
711:        non_rhel_kernels = get_installed_pkgs_w_different_fingerprint(system_info.fingerprints_rhel, "kernel")
823:    installed_rhel_kernels = get_installed_pkgs_by_fingerprint(system_info.fingerprints_rhel, name="kernel")
829:    non_rhel_kernels = get_installed_pkgs_w_different_fingerprint(system_info.fingerprints_rhel, "kernel*")

convert2rhel/unit_tests/pkghandler_test.py
1831:    monkeypatch.setattr(system_info, "fingerprints_rhel", ["rhel_fingerprint"])
```

Reply:
Agreed! But we can do that in a follow-up PR